lib/src/util/derive_associated_token_account_address.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:cryptography/cryptography.dart' as crypto;
+import 'package:solana/src/base58/base58.dart' as base58;
+import 'package:solana/src/constants/constants.dart';
+
+const _nonce = 255;
+final _sha256 = crypto.Sha256();
+const _magicWord = 'ProgramDerivedAddress';
+
+Future<String> deriveAssociatedTokenAddress({
```

Comment:
Please add some documentation to it. It would be good to have a description of why it's needed at all.